installer/pkg/cmd/service_catalog.go

```diff
@@ -524,9 +525,24 @@ func checkDependencies() error {
 	if len(missingCmds) > 0 {
 		return fmt.Errorf("%s commands not found in the PATH", strings.Join(missingCmds, ","))
 	}
+
+	// Also print out current account, project and zone information.
+	configs, err := gcp.GetConfigMap()
+	if err != nil {
+		return fmt.Errorf("error retrieving gcloud config: %v", err)
+	}
+
+	fmt.Printf("account: %s\n", getValueFromConfigMap("core", "account", configs))
+	fmt.Printf("project: %s\n", getValueFromConfigMap("core", "project", configs))
+	fmt.Printf("zone: %s\n", getValueFromConfigMap("compute", "zone", configs))
+
 	return nil
 }
 
+func getValueFromConfigMap(section, property string, configs map[string]interface{}) string {
+	return configs[section].(map[string]interface{})[property].(string)
```

Comment:
any possible run-time error when accessing `section` and `property`?

Please do not make the binary panic like `panic: interface conversion: interface {} is nil, not string`

Reply:
Good point. So do you mean that in general, we don't want to do `.(<type>)`, or we want to use some defensive coding logic to prevent panic?

Reply:
either works fine.

`a, ok := b.(string)` and check `ok`, or
`a, ok := b[c]`and check `ok`.